src/frontend/js/components/Tabs/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button } from '@openfun/cunningham-react';
+import classNames from 'classnames';
+import {
+  Dispatch,
+  PropsWithChildren,
+  SetStateAction,
+  createContext,
+  useContext,
+  useMemo,
+  useState,
+} from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Maybe } from 'types/utils';
+
+interface TabsContextProps {
+  activeTabName?: string;
+  setActiveTabName?: Dispatch<SetStateAction<Maybe<string>>>;
+}
+const TabsContext = createContext<TabsContextProps>({});
+
+const Tabs = ({ children }: PropsWithChildren) => {
+  const [activeTabName, setActiveTabName] = useState<string>();
+  const contextValue = useMemo(() => {
+    return { activeTabName, setActiveTabName };
+  }, [activeTabName]);
+
+  return (
+    <div className="tabs">
+      <TabsContext.Provider value={contextValue}>{children}</TabsContext.Provider>
+    </div>
+  );
+};
+
+Tabs.Header = ({ children }: PropsWithChildren) => {
+  return <div className="tabs__header">{children}</div>;
+};
+
+Tabs.Content = ({ children }: PropsWithChildren) => {
+  return <div className="tabs__content">{children}</div>;
+};
+
+interface TabProps extends PropsWithChildren {
+  name: string;
+  href?: string;
+  onClick?: () => void;
+  initialIsActive?: boolean;
```

Comment:
IMO, to set the initial active tab, we should do it at the context level not here.

Reply:
I didn't feel the need to create an custom provider, it's feel overkill to me but i'll do it.